drf_spectacular/views.py

```diff
@@ -11,10 +13,17 @@
 )
 from drf_spectacular.settings import spectacular_settings
 from drf_spectacular.types import OpenApiTypes
-from drf_spectacular.utils import extend_schema
+from drf_spectacular.utils import OpenApiParameter, extend_schema
 
 if spectacular_settings.SERVE_INCLUDE_SCHEMA:
     SCHEMA_KWARGS: Dict[str, Any] = {'responses': {200: OpenApiTypes.OBJECT}}
+
+    if settings.USE_I18N:
+        SCHEMA_KWARGS['parameters'] = [
+            OpenApiParameter(
+                'lang', str, OpenApiParameter.QUERY, enum=list(dict(settings.LANGUAGES).keys())
```

Comment:
sort the lang keys?

currently I get the same order as my setting, which is not alpha ordered

Reply:
opted against. imho the user should decide which order is important to him.